Cluster group volumes backup (#1140)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/05-backup-restore.md

Go to **Project Settings** > **Backup locations** > **Add a New Backup location**.

# Create a Cluster Backup

> Backups can be scheduled or initiated in an on-demand basis during cluster creation as well as can be scheduled for a running cluster. The following information is required for configuring a cluster backup:

Reviewer: ```suggestion
Backups can be scheduled or initiated on demand during cluster creation. Backups can also be scheduled for a running cluster. The following information is required to configure a cluster backup:
```
